### DISCUSSION TITLE
mcp has a forum now!

### DISCUSSION THREAD

**Discussion by lindeloev** [lindeloev/mcp#98] [General]
👋 Welcome! Feel free to ask questions and share your experiences in this forum!

You can also ask on [StackOverflow](https://stackoverflow.com) or [CrossValidated](https://stats.stackexchange.com) where I'm monitoring the [[change-point]](https://stackexchange.com/search?q=%5Bchange-point%5D) tag. And of course, [raise an Issue](https://github.com/lindeloev/mcp/issues) right here on the Github repo for more specific feature requests or bug reports.

**Comment by JFormoso** [comment]


**Reply by lindeloev** [reply]
Hi @JFormoso !

Please make a new Discussion thread and we'll see what we can do there :-) In your post, say a bit about *how* different your outputs are and whether you just want *approximately* the same results (e.g. parameters estimated to 0.1% precision) or *exactly* the same results.
